sparse/tests/test_coo.py

```diff
@@ -894,6 +894,7 @@ def test_scipy_sparse_interface():
 
     assert_eq(x, xx)
     assert_eq(x.T.dot(x), xx.T.dot(xx))
+    assert isinstance(x + xx, (COO, scipy.sparse.spmatrix))
```

Comment:
Perhaps test _only_ for `COO`? Since we're "higher level" than `scipy.sparse`, it makes sense that mixed binary operations don't return `scipy.sparse.spmatrix`.

Reply:
In addition (just as a minor quip) I would parametrize this with `format` for scipy.sparse and check for (at least) `csr`, `csc`, `coo` and `dok`.

Reply:
Done